cointop/config.go

```diff
@@ -227,9 +227,9 @@ func (ct *Cointop) ConfigToToml() ([]byte, error) {
 		if !ok || entry.Coin == "" {
 			continue
 		}
-		var amount string = strconv.FormatFloat(entry.Holdings, 'f', -1, 64)
-		var coinName string = entry.Coin
-		var tuple []string = []string{coinName, amount}
+		var amount = strconv.FormatFloat(entry.Holdings, 'f', -1, 64)
```

Comment:
suggestion: use shorthand

```go
amount := ...
coinName := ...
tuple := ...
```

Reply:
Seems like lots of places we could do that? Happy to replace 'em all?